buildconfig/pygame-stubs/time.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Optional, Union
+from pygame.event import Event
+
+def get_ticks() -> int: ...
+def wait(milliseconds: int) -> int: ...
+def delay(milliseconds: int) -> int: ...
+def set_timer(eventid: Union[int, Event], milliseconds: int, once: Optional[bool]=False) -> None: ...
```

Comment:
time.c@time_set_timer set_timer doesn't use keyword arguments.

I guess `eventid` should just be called `event`, or `event_type`.